### DISCUSSION TITLE
Increase the time span for logic switches

### DISCUSSION THREAD

**Discussion by 3DRacer** [EdgeTX/edgetx#3341] [Ideas]
The duration of the logic switches can be programmed from 0 to 25 seconds. This period of time can be too short for some applications. Would it be possible to extend this time? In my case, 0 to 60 seconds would be sufficient.

**Comment by elecpower** [comment]
@3DRacer to support your position can you please add a real life example.

**Reply by 3DRacer** [reply]
I mounted a momentary tilt switch in my TX16S (as SJ). With this and my arming switch I created an OR Logic switch to switch on the LCD lighting when tilting the transmitter or disarming my copter. I set the duration to 25 seconds. I would like to extend this time a bit. The other control options of the LCD do not correspond to my imagination.
